src/pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover.js

```diff
@@ -36,6 +36,7 @@ const policySelector = (policy) =>
     policy && {
         type: policy.type,
         role: policy.role,
+        isPolicyExpenseChatEnabled: policy.isPolicyExpenseChatEnabled,
```

Comment:
With the current state, `policy.isPolicyExpenseChatEnabled` may be missing, so here you will define `isPolicyExpenseChatEnabled: undefined`

It is easier for types like this to keep them as `true`/`false`, instead of `true`/`false`/`undefined`

Reply:
> so here
Did you miss adding a link for this? I'm not sure where to set it `undefined`

Reply:
ah, I mean that `policy.isPolicyExpenseChatEnabled` was possibly missing, but not anymore since the changes you did in Web-E.. so you can ignore this.